kinbot/mess.py

```diff
@@ -367,24 +429,27 @@ def write_bimol(self, prod_list, well_add, freqFactor, uq_iter, bless=0):
             combined_freq = ''
             combined_hir = ''
         smi = []
-        for nsp, species in enumerate(prod_list):
+        for i, species in enumerate(prod_list):
             smi.append(species.smiles)
             if species.natom > 1:
-
                 if self.par['pes']:
                     name = '{{fr_name_{}}}'.format(species.chemid)
+                    freq = '{{freq_{}}}'.format(i) 
+                    nfreq = '{{nfreq_{}}}'.format(i)
```

Comment:
When PES is turned on the `freq` and `nfreq` values become keywords to fill in during the final PES MESS file creation step.